instrumentation/mongodb/src/main/java/brave/mongodb/TraceMongoCommandListener.java

```diff
@@ -61,11 +61,13 @@
    * all callbacks happen on the same thread.
    */
   @Override public void commandStarted(CommandStartedEvent event) {
+    String databaseName = event.getDatabaseName();
+    if ("admin".equals(databaseName)) return; // don't trace commands like "endSessions"
```

Comment:
super strange that in a patch "admin" database commands suddenly show up. Due to order of things, we would only see "endSessions"! This explicitly gets rid of this similar to p6spy until/unless someone asks for full admin coverage and in that case end w/o start probably is undesirable.